hadoop-ozone/dev-support/checks/kubernetes.sh

```diff
@@ -13,9 +13,37 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 cd "$DIR/../../.." || exit 1
 
+if [[ "$(uname -s)" = "Darwin" ]]; then
+  echo "k3s is not supported on Mac" >&2
+  exit 1
+fi
+
+export KUBECONFIG=/etc/rancher/k3s/k3s.yaml
+
+_install_tool_callback() {
+  # Install k3s
+  curl -sfL https://get.k3s.io | sh -
+
+  sudo chmod a+r $KUBECONFIG
+
+  # Install flekszible
+  wget https://github.com/elek/flekszible/releases/download/v1.8.1/flekszible_1.8.1_Linux_x86_64.tar.gz -O - | tar -zx
+  chmod +x flekszible
+  mkdir -p ${TOOL_DIR}/bin
+  mv -iv flekszible ${TOOL_DIR}/bin/
+
+  # Install robotframework
+  sudo pip install robotframework
```

Comment:
You can use something like this to avoid sudo:

```
 virtualenv "${TOOL_DIR}"/robotenv
 source "${TOOL_DIR}"/robotenv/bin/activate
 pip install robotframework
```
and you can add `"${TOOL_DIR}"/robotenv/bin` to the path